### PR TITLE
[MBL-15936][Student] Canvas for Elementary theme is applied to courses which don't have it enabled.

### PR DESCRIPTION
**Pull request by tamaskozmer**
Test plan: Description in the ticket. There are also more cases where this bug could happen. It can also happen when opening a non K5 course grades from the K5 grades page. In that case we should redirect to the standard grades page.

refs: MBL-15936
affects: Student
release note: Fixed a bug where elementary subject page would open instead of course page.

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

File: apps/student/src/main/java/com/instructure/student/features/elementary/course/ElementaryCourseFragment.kt [hunk 1]
```diff
@@ -92,6 +97,12 @@ class ElementaryCourseFragment : Fragment() {
 
             }
         })
+
+        viewModel.events.observe(viewLifecycleOwner, Observer { event ->
```
Comment (hermannakos) [inline apps/student/src/main/java/com/instructure/student/features/elementary/course/ElementaryCourseFragment.kt:101]: There's a weird flashing when redirecting to the standard subject page. If we move this and the data fetching to the onCreateView could that perhaps make it a bit smoother?

**Review by hermannakos — APPROVED** [review]

**Pull request merged**